packages/core/src/components/ConfirmationModal/Stories/ConfirmationModalExample.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+
+import { select } from '@storybook/addon-knobs';
+
+import { Button, Icon } from 'patternfly-react';
+import ConfirmationModal from '../ConfirmationModal';
+
+class ConfirmationModalExample extends Component {
+  state = {
+    show: false
+  };
+
+  onCancel = () => {
+    this.setState(() => ({ show: false }));
+  };
+
+  onConfirm = () => {
+    // Do some stuff
+    this.setState(() => ({ show: false }));
+  };
+
+  showModal = () => {
+    this.setState(() => ({ show: true }));
+  };
+
+  render() {
+    // *************************************************************************
+    // * StoryBook Knobs Config
+    // *************************************************************************
+    const value = select(
+      'Icon',
+      ['info', 'error-circle-o', 'warning-triangle-o'],
+      'info'
+    );
+
+    const body = (
+      <React.Fragment>
+        <p>You are about to do something</p>
+        <ul>
+          <li>Here are some possible</li>
+          <li>consequences of continuing</li>
+          <li>with your action</li>
+        </ul>
+      </React.Fragment>
+    );
+
+    return (
+      <React.Fragment>
+        <Button onClick={this.showModal} bsStyle="primary">
+          Launch Confirmation Modal
+        </Button>
+        <ConfirmationModal
+          show={this.state.show}
+          title="Modal Title"
+          icon={<Icon type="pf" name={value} />}
+          heading="Body Heading"
+          body={body}
+          onCancel={this.onCancel}
+          onConfirm={this.onConfirm}
+        />
+      </React.Fragment>
+    );
+  }
+}
+
+export default ConfirmationModalExample;
+
+export const ConfirmationModalExampleSource = `
+import React, { Component } from 'react';
+
+import { select } from '@storybook/addon-knobs';
+
+import { Button, Icon } from 'patternfly-react';
+import ConfirmationModal from '../ConfirmationModal';
+
+class ConfirmationModalExample extends Component {
+  state = {
+    show: false
+  };
+
+  onCancel = () => {
+    this.setState(() => ({ show: false }));
+  };
+
+  onConfirm = () => {
+    // Do some stuff
+    this.setState(() => ({ show: false }));
+  };
+
+  showModal = () => {
+    this.setState(() => ({ show: true }));
+  };
+
+  render() {
+    // *************************************************************************
+    // * StoryBook Knobs Config
+    // *************************************************************************
+    const value = select(
+      'Icon',
+      ['info', 'error-circle-o', 'warning-triangle-o'],
+      'info'
+    );
+
```

Comment:
can we add knobs for setting the title and heading as well as including the body or not?